CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ Our versioning scheme is `YEAR.N` where `N` is incremented whenever a new releas
 
 ## [??? (unreleased)]
 
+### Added
+
+- Ostree update on IP Secondaries: [PR](https://github.com/advancedtelematic/aktualizr/pull/1518)
+- Basic file update on IP Secondaries: [PR](https://github.com/advancedtelematic/aktualizr/pull/1518)
```

Comment:
I would argue that the OSTree item here should be under "Fixed", as it has been supported for a while. Maybe it's worth saying something like "Fixed several issues with OSTree updates on IP Secondaries".

Reply:
ok